metricbeat/module/kubernetes/util/prometheus.go

```diff
@@ -0,0 +1,48 @@
+package util
+
+import (
+	"fmt"
+
+	"github.com/elastic/beats/metricbeat/helper"
+
+	dto "github.com/prometheus/client_model/go"
+	"github.com/prometheus/common/expfmt"
+)
+
+func GetFamilies(http *helper.HTTP) ([]*dto.MetricFamily, error) {
```

Comment:
ok to the helper, prometheus collector issue explained in a previous comment, but I can update it to share the helper